client/plots/profileBarchart.js

```diff
@@ -41,22 +41,23 @@ class profileBarchart {
 		this.data = await this.app.vocabApi.getAnnotatedSampleData({
 			terms: twLst
 		})
-		this.regions = ['']
-		this.incomes = ['']
-		this.countries = ['']
-
-		this.region = this.config.region || this.regions[0]
-		this.income = this.config.income || this.incomes[0]
-		this.country = this.config.country || this.countries[0]
+		this.regions = [{ key: 'Global', label: 'Global' }]
+		this.incomes = ['Global']
```

Comment:
there's no `Global` for income group, this seems to cause trouble when switching dropdown

Reply:
the income does not require key and label, is ok to pass just 'Global'